tests/org/tests_get_group.py

```diff
@@ -0,0 +1,48 @@
+from org.models import *
+from tests.AuthAPITestCase import AuthAPITestCase
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+from users.models import User
+from org.serializers import CreateOrgSerializer
+from org.custom_model_field import PermissionField as Permissions
+import uuid
+
+
+class EditOrgAPITestCase(AuthAPITestCase):
```

Comment:
Since it about viewing the Org details and not editing it I suppose this isn't the most appropriate way of naming it. But Since it is also a part of the `EditOrgView`, it might seem relatively correct too. **A docstring clarifying the purpose** of this particular Test Case seems appropriate to me. Any suggestions for the same?

Also since this isn't related to groups I suppose the file name `tests_get_group.py`  must be renamed.